### PR TITLE
status bar height and mouse icon size, fixes #1026, #1303, #1368

### LINKED ISSUE #1026: status bar is resized during showMessage()
### Expected behavior
The status bar height is fixed

### Observed behavior
During showMessage(), e.g. on auto save, the status bar height is reduced.
This makes the layout jump on every auto save cycle.

### Steps to reproduce or sample file
Open a new file
Draw some entities
Wait for auto save

For faster success reduce the auto save timer to minimum, means 1 minute

### Operating System and LibreCAD version info
Ubuntu 18.10
LibreCAD 2.1.2

see forum thread http://forum.librecad.org/cursor-jumps-during-autosave-tp5716571.html for more details

### PR DESCRIPTION
**Pull request by melwyncarlo**
* Solved issue #1368
* Solved issue #1303 
* Solved issue #1026 

<hr>

**You may ignore this now :** 
> The icon's size is implemented as is. The big mouse icon (located at `librecad/res/icons`) size was **80x80** pixels. The small mouse icon (located at `librecad/res/ui`) size is **24x24** pixels. I think _this_ icon was used before. To implement the right icon, just ensure that the icon size is about **25x25** pixels.

### CODE REVIEW

**Comment by melwyncarlo** [comment]
I figured out the actual problem: how to scale the image via the program code, and not the physical image file.

The QPixmap class has a function called `scaled` that does the image rescaling.
I have not preserved the aspect ratio. If you wish to preserve the aspect ratio, 
then I could append a third argument to the function: `Qt::KeepAspectRatio`

**Comment by melwyncarlo** [comment]
**Is this better?**

<hr>

image

image

**Comment by melwyncarlo** [comment]
My v1 commit does not cause the status bar to change height unexpectedly. But the mouse icon is too small.

My v2 commit allows for the SVG mouse icon to be scaled and displayed, but that causes the status bar to change height unexpectedly.

I tried converting the `setMinimumHeight` function of the status bar to `setFixedHeight` instead; yet no relief. I think that the problem is an indirect effect of something else.

**Comment by melwyncarlo** [comment]
My v3 commit does it best. The `qg_mousewidget.cpp` has been reset back to its **master** branch version.

All changes have been concentrated in the file `qc_applicationwindow.cpp`.

File: librecad/src/main/qc_applicationwindow.cpp [hunk 1]
```diff
@@ -3226,9 +3234,12 @@ void QC_ApplicationWindow::widgetOptionsDialog()
         settings.setValue("AllowStatusbarHeight", allow_statusbar_height);
         if (allow_statusbar_height)
         {
-            int statusbar_height = dlg.statusbar_height_spinbox->value();
+            statusbar_height = dlg.statusbar_height_spinbox->value();
             settings.setValue("StatusbarHeight", statusbar_height);
-            statusBar()->setMinimumHeight(statusbar_height);
+            statusBar()->setFixedHeight(statusbar_height);
+
+            mouseWidget->findChild<QLabel*>("lMousePixmap")->setPixmap(
```
Comment (melwyncarlo) [inline librecad/src/main/qc_applicationwindow.cpp:3241]: This is *BAD* (commit v3). It basically rescales the pixmap image, but relatively. So after scaling down, and then rescaling up, it rasterizes the image. 

Fixed in commit v4.

**Comment by lordofbikes** [comment]
fixed with eb0223968761df405c16a4885fff36c2fd05e9d1

**Pull request closed without merging**